composer.json

```diff
@@ -13,7 +13,7 @@
         "http-interop/http-factory-guzzle": "^1.0",
         "knplabs/github-api": "^3.4",
         "m4tthumphrey/php-gitlab-api": "^11.5",
-        "nikic/php-parser": "^4.13",
+        "nikic/php-parser": "4.13.*",
```

Comment:
Can we lock on minor?

Reply:
Yes, here we locked on minor

Reply:
Unfortunately, ^4.13.2 moved to 4.14.0 during composer update

Reply:
Hi @VladislavStrelchenko,
I hope that minor update shouldn't cause any issues.
Otherwise, could you please investigate it?

Reply:
I created dedicated issue https://spryker.atlassian.net/browse/PPLUS-1384
Also fixed in the commit https://github.com/spryker-sdk/upgrader/pull/42/commits/d7986f93cc25e3b9cf964513b5d21a5a76c37533

Reply:
In the case with this fix, Class_exist() doesn't see Spryker classes